src/main/java/net/sf/jabref/gui/JabRefFrame.java

```diff
@@ -2253,4 +2261,28 @@ public void setPreviewToggle(boolean enabled) {
     public PushToApplications getPushApplications() {
         return pushApplications;
     }
+
+
+    private class UndoRedoEventManager {
+
+        @Subscribe
+        public void listen(UndoRedoEvent event) {
+            updateTexts(event);
+            JabRefFrame.this.getCurrentBasePanel().updateEntryEditorIfShowing();
```

Comment:
Is this really necessary? I think the entry editor listens for changes to the displayed entry, right?